sdcm/nemesis.py

```diff
@@ -1912,11 +1917,11 @@ def disrupt_grow_shrink_cluster(self):
         for _ in range(add_nodes_number):
             self.set_target_node()
             self.log.info("Next node will be removed %s", self.target_node)
-            self.metrics_srv.event_start('del_node')
             try:
+                InfoEvent(message='StartEvent - ShrinkCluster started decommissioning a node')
                 self.cluster.decommission(self.target_node)
             finally:
-                self.metrics_srv.event_stop('del_node')
+                InfoEvent(message='FinishEvent - ShrinkCluster has done decommissioning a node')
```

Comment:
here we should also wait some 3 or 5 minutes between each decommission

Reply:
why?

Reply:
ah, it runs with decommission... so never mind.